### ISSUE TITLE
prototype ALIGNN ST(E)M model

### ISSUE THREAD

**Issue by bdecost** [usnistgov/atomvision#5]
Initial task: Bravais lattice classification?

Dataloader:
- [x] pretrained U-Net (from #3)
- [x] DGLGraph constructor 
- [ ] Detected atom featurization (intensity histograms? guess at species?)

model:
- [ ] ALIGNN{2,2} with default parameters otherwise?
- [ ] mask out atoms on the edge of the FoV during global pooling?

**Comment by bdecost** [comment]
This prototype is mostly finished. The guess at actual species is still an open question, and building the line graph is a next step but shouldn't be as open ended as guessing the species
